modules/aruco/misc/pattern_generator/README.md

```diff
@@ -0,0 +1,40 @@
+# OpenCVMarkerPrinter
+
+## Description
+This small app can save some commonly used opencv markers such as ArUco, ArUcoGrid, Chessboard and ChArUco to vector graphics file. **Supported vector graphics file format: .svg, .pdf and .ps. Supported image file format: .png (not recommend).**
+
+<img src="./doc/images/0001.jpg" height="160" />
+
+### Dependencies
+#### MarkerPrinter
+  * numpy
```

Comment:
No problem~

Reply:
@mshabunin How to determine minimal dependencies versions?

Reply:
In my environment (WIN10, python3.6),  

it cannot directly install pycairo and CairoSVG via pip 
(it will report some error message.... orz), 

so I install pre-build pycairo and cairocffi  wheel from https://www.lfd.uci.edu/~gohlke/pythonlibs/, 
and than install CairoSVG via pip.....

I will find out a stable version that can be installed via pip and add to requirements.txt.

Reply:
Ok, leave it as-is then.